.github/ISSUE_TEMPLATE/question.yml

```diff
@@ -0,0 +1,16 @@
+name: "🙋 Question"
```

Comment:
[note] we don't really need `ISSUE_TEMPLATE`, because they take effect from the `default` branch. But I see why this came in - as this was part of our initial CI efforts. *No need to remove this because of my comment*, just a comment. Keeping this may be good if we decide to cut `main` -> `slash` early next year.